more/free-programming-cheatsheets.md

```diff
@@ -202,6 +203,11 @@
 * [SEO CheatSheet](https://learn-the-web.algonquindesign.ca/topics/seo-cheat-sheet/) - Algonquin Design (HTML)
 
 
+#### Tailwind CSS
+
+* [Tailwind CSS Cheat Sheet](https://tailwindcomponents.com/cheatsheet) - tailwind**components**
```

Comment:
```suggestion
* [Tailwind CSS Cheat Sheet](https://tailwindcomponents.com/cheatsheet) - Tailwind Components
```
It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category. In this case, just put it in the HTML and CSS category.

Reply:
I thought of adding more Tailwind Cheat sheets in the Future, but as of now it would be better under the HTML and CSS category